### DISCUSSION TITLE
How do I claim pets?

### DISCUSSION THREAD

**Discussion by Maple38** [GriefPrevention/GriefPrevention#822] [Q&A] [answered]
I couldn't figure out how I'm supposed to claim an animal. Could you please explain? (also it would be useful if you put it on your website in case another stupid person like me comes along).

**Comment by RoboMWM** [comment, marked as answer]
you don't claim animals. Untamable animals "belong" to whichever claim they're in. Tameable animals "belong" to the player that tamed them.
